ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```

Comment:
@PeterDaveHello Or maybe I can ask the author if he can write an English description?